torchgeo/datasets/ssl4eo.py

```diff
@@ -0,0 +1,250 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Self-Supervised Learning for Earth Observation."""
+
+import os
+import random
+from typing import Callable, Dict, Optional, cast
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import check_integrity, extract_archive
+
+
+class SSL4EOS12(NonGeoDataset):
+    """SSL4EO-S12 dataset.
+
+    The `Self-Supervised Learning for Earth Observation - Sentinel-1/2
+    <https://github.com/zhu-xlab/SSL4EO-S12>`_ dataset is a large-scale mutilmodal
```

Comment:
"multimodal" typo

Reply:
Submitted a PR to fix upstream README where I copied this from.